### DISCUSSION TITLE
Looking for an old wallpaper

### DISCUSSION THREAD

**Discussion by MrSmileFrog** [rocksdanister/lively#2126] [Q&A] [answered]
I remember in early versions there were built-in wallpapers, such a red stripe that moved under the frequencies when playing sounds / music.  Maybe someone has this file? I really need it. I searched, but unfortunately I couldn't find it either in the old versions or in the wallpaper collections.
![WSJDCfpFie](https://github.com/rocksdanister/lively/assets/119322786/a48affc0-ff54-4766-a572-3eb1bf7e5faf)
As in this image approximately. And with full squares.

Edit:    https://github.com/rocksdanister/audio-visualizer-wallpaper/releases/download/v1.0.0.1/Simple.visualizer.zip

**Comment by rocksdanister** [comment, marked as answer]
If it came with the app then its probably Simple Visualizer:
https://github.com/rocksdanister/audio-visualizer-wallpaper

**Reply by MrSmileFrog** [reply]
Yeah, thank you very much!!>
